src/sentry/templates/sentry/emails/crons/broken-monitors.html

```diff
@@ -0,0 +1,34 @@
+{% extends "sentry/emails/base.html" %}
+
+{% load i18n %}
+{% load sentry_helpers %}
+{% load sentry_assets %}
+
+{% block head %}
+    {{ block.super }}
+    <style>
+        img.hero {margin-bottom: 15px; width: 100%;}
+    </style>
+{% endblock %}
+
+{% block main %}
+    <img class="hero" src="{% absolute_asset_url 'sentry' 'images/email/crons/crons-hero.jpg' %}" />
+    <h3>Your Cron Monitors Aren't Working</h3>
+    <p>
+      We've noticed the cron monitors below have not processed a successful check-in for multiple days:
+    </p>
+      <ul>
+        {% for monitor_name, monitor_url, earliest_incident_start_date in broken_monitors %}
+          <li>
+            <a href="{{monitor_url}}">{{monitor_name}}</a> | Failing since {{earliest_incident_start_date|date:"N j, Y, g:i:s a e"}}
+          </li>
```

Comment:
IMO this would look better if we put the `failing since` on a second line. So instead of a `|` a `<br />`.

wdyt @davidenwang

Reply:
i like it, changing now